<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="INDLAPTOP403" name="com.arcfacalities.common.TC023_GET_Toget_S3presigned_URLtodownloadPDF_fromS3attachediththeForm" tests="1" failures="0" timestamp="6 Dec 2019 06:45:41 GMT" time="0.000" errors="1">
  <testcase name="presignedURt0downloadPDF" time="0.000" classname="com.arcfacalities.common.TC023_GET_Toget_S3presigned_URLtodownloadPDF_fromS3attachediththeForm">
    <error type="org.testng.TestNGException" message="
Method public void com.arcfacalities.common.TC023_GET_Toget_S3presigned_URLtodownloadPDF_fromS3attachediththeForm.presignedURt0downloadPDF(java.util.Hashtable) throws java.lang.Exception requires a @DataProvider named : presignedURt0downloadPDF">
      <![CDATA[org.testng.TestNGException: 
Method public void com.arcfacalities.common.TC023_GET_Toget_S3presigned_URLtodownloadPDF_fromS3attachediththeForm.presignedURt0downloadPDF(java.util.Hashtable) throws java.lang.Exception requires a @DataProvider named : presignedURt0downloadPDF
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:274)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1300)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1015)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1104)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- presignedURt0downloadPDF -->
</testsuite> <!-- com.arcfacalities.common.TC023_GET_Toget_S3presigned_URLtodownloadPDF_fromS3attachediththeForm -->
